packages/@react-spectrum/badge/src/Badge.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {classNames, SlotProvider, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import {DOMRef} from '@react-types/shared';
+import {filterDOMProps} from '@react-aria/utils';
+import React, {forwardRef} from 'react';
+import {SpectrumBadgeProps} from '@react-types/badge';
+import styles from '@adobe/spectrum-css-temp/components/badge/vars.css';
+import {Text} from '@react-spectrum/text';
+import {useProviderProps} from '@react-spectrum/provider';
+
+
+function Badge(props: SpectrumBadgeProps, ref: DOMRef<HTMLDivElement>) {
+  let {
+    children,
+    isDisabled,
+    variant,
+    ...otherProps
+  } = useProviderProps(props);
+  let domRef = useDOMRef(ref);
+  let {styleProps} = useStyleProps(otherProps);
+  let isTextOnly = React.Children.toArray(props.children).every(c => !React.isValidElement(c));
+
+  return (
+    <span
+      {...filterDOMProps(otherProps)}
+      {...styleProps}
+      role="presentation"
+      className={classNames(
+        styles,
+        'spectrum-Badge',
+        'spectrum-Badge--fixed-top',
```

Comment:
not a real classname? i think at one point we called them `anchorTop` this should be `fixedTop` if that's what it's meant to be

Reply:
yeah i had the fixed prop in there but removed it in favor of a later version. forgot to remove this from the code